src/services/queryUserIntput.service.ts

```diff
@@ -0,0 +1,21 @@
+import { POI } from "../types/main";
```

Comment:
please follow the file name convention we've had up until now. Rename the file to: query-user-input.service.ts.